packages/block-editor/src/components/block-toolbar/style.scss

```diff
@@ -285,3 +285,19 @@
 		}
 	}
 }
+
+.block-editor-block-toolbar-change-design-content-wrapper {
+	padding: $grid-unit-15;
+	width: 320px;
+	.block-editor-block-patterns-list {
```

Comment:
What's your current thoughts about absorbing this into the component with `variant="grid"` or something. How feasible...?

Reply:
I think it's doable (regarding the current usages and wrappers etc..). It's on my list to do this and see to consolidate the similar replace lists.